src/redux/actions/instructorActions.js

```diff
@@ -1 +1,54 @@
 // Declare Actions
+import axios from 'axios';
+
+export const ERROR_ACTION = 'ERROR';
+export const GET_USER_ACTION = 'GET_USERS';
+export const GET_COURSES_ACTION = 'GET_COURSES';
+export const GET_INBOX_ACTION = 'GET_INBOX';
+
+export const getusers = () => async dispatch => {
```

Comment:
We should make sure that the casing for the function names is the same. Just to stay conventional.